pkgs/apps/wannier90/default.nix

```
@@ -0,0 +1,49 @@
+{ stdenv, lib, gfortran, blas, lapack, python2, fetchFromGitHub }:
+
```

I assume that the package can only handle 4 byte integers. Let's ensure we build it with the correct API.
```suggestion
assert (!blas.isILP64);
assert blas.isILP64 == lapack.isILP64;

```